tests/Unit/ApplicationPasswordTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Roots\PasswordBcrypt\Tests\Unit;
+
+use Roots\PasswordBcrypt\Tests\TestCase;
+use Roots\PasswordBcrypt\Tests\Constants;
+use Mockery;
+
+use function Brain\Monkey\Functions\expect;
+use function Brain\Monkey\Filters\expectApplied;
+
+class ApplicationPasswordTest extends TestCase
+{
+
+    /** @test */
+    public function phpass_application_passwords_should_be_verified_and_converted_to_bcrypt()
```

Comment:
This is a gnarly test but couldn't think of a better way without full integration tests.